platform/migration/platform-migratable-util/src/main/java/org/codice/ddf/migration/util/MigratableUtil.java

```diff
@@ -192,6 +192,24 @@ public void copyFileFromJavaPropertyValue(@NotNull Path propertyFilePath,
                         warnings));
     }
 
+    /**
+     * Reads a java properties file and returns the value of the specified property.
+     * 
+     * @param propertyFilePath path to the Java properties file that contains the path to the
+     *                         source file to copy
+     * @param javaProperty     name of the property inside the Java properties file that contains
+     *                         the path to the source file
+     * @return                 the value of the property if found, null otherwise
+     * @throws MigrationException
```

Comment:
We should mention when this exception will be thrown so callers know what to expect.